cluster-http/src/test/scala/akka/cluster/http/management/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -275,5 +286,91 @@ class ClusterHttpManagementRoutesSpec
         }
       }
     }
+
+    "return shard region details" when {
+
+      "calling GET /shard_regions/{name}" in {
+        val config = ConfigFactory.parseString(
+          """
+            |akka.cluster {
+            |  auto-down-unreachable-after = 0s
+            |  periodic-tasks-initial-delay = 120 seconds // turn off scheduled tasks
+            |  publish-stats-interval = 0 s # always, when it happens
+            |  failure-detector.implementation-class = akka.cluster.FailureDetectorPuppet
+            |  sharding.state-store-mode = ddata
+            |}
+            |akka.actor.provider = "cluster"
+            |akka.remote.log-remote-lifecycle-events = off
+            |akka.remote.netty.tcp.port = 0
+          """.stripMargin
+        )
+        val configClusterHttpManager = ConfigFactory.parseString(
+          """
+            |akka.cluster.http.management.hostname = "127.0.0.1"
+            |akka.cluster.http.management.port = 20100
```

Comment:
@patriknw i couldn't get the dependencies to work. 

First I added  this to library dependencies

`"com.typesafe.akka" %% "akka-stream-testkit"                % AkkaVersion     % "test"`

But that didn't make `TestUtils` accessible since its in the `test` configuration of `akka-stream-testkit` [source](https://github.com/akka/akka/blob/9d2bec7f232b628cc087231af75e457072823e61/akka-stream-testkit/src/test/scala/akka/stream/testkit/TestUtils.scala)

So then I tried

`"com.typesafe.akka" %% "akka-stream-testkit"                % AkkaVersion     % "test->test"`

And got this error:
```
[error] (cluster-http/*:update) sbt.ResolveException: unresolved dependency: com.typesafe.akka#akka-stream-testkit_2.11;2.4.18: configuration not public in com.typesafe.akka#akka-stream-testkit_2.11;2.4.18: 'test'. It was required from com.lightbend.akka#akka-management-cluster-http_2.11;0.2+11-7994a5b9+20170531-1122 test
```

let me know if you have any other ideas on how to do this. 
Note that the `ClusterHttpManagementSpec` also hardcodes in ports (which I agree is a bad pattern)